comment.go

```diff
@@ -298,6 +328,27 @@ func (f *File) addComment(commentsXML, cell string, formatSet *formatComment) {
 	f.Comments[commentsXML] = comments
 }
 
+// delComment provides a function to delete comment
+func (f *File) delComment(path, cell string) {
+	if comments := f.commentsReader(path); comments != nil {
+		for k, v := range comments.CommentList.Comment {
+			if v.Ref == cell {
+				l := len(comments.CommentList.Comment)
+				if l > 1 {
+					comments.CommentList.Comment = append(
+						comments.CommentList.Comment[:k],
+						comments.CommentList.Comment[k+1:]...,
+					)
+				} else {
```

Comment:
In my personal approach of choosing one is that using continue statement instead of else.